src/main/steps/interstitial/content.ts

```diff
@@ -0,0 +1,18 @@
+import { TranslationFn } from '../../app/controller/GetController';
+
+const en = {
+  title: 'Interstitial',
```

Comment:
I don't think this will be needed as this interstitial page will be implemented by https://github.com/hmcts/nfdiv-frontend/pull/323